pages/common/hadolint.md

```diff
@@ -0,0 +1,16 @@
+# hadolint
+
+> Lints Dockerfiles to help you build [best practice](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices) Docker images.
+> More information: https://github.com/hadolint/hadolint.
+
+- Lint a Dockerfile, printing any output to STDOUT:
+
+`hadolint {{Dockerfile}}`
+
+- Lint a Dockerfile ignoring specific rules and outputting JSON:
+
+`hadolint --ignore {{DL3006}} --format json {{Dockerfile}}`
+
+- Lint multiple Dockerfiles while specifying a trusted registry:
+
+`hadolint --trusted-registry {{my.company.com:500}} {{app/Dockerfile}} {{db/Dockerfile}}`
```

Comment:
We use `example.com` for examples, the port is required?
```suggestion
`hadolint --trusted-registry {{example.com}}:{{500}} {{path/to/Dockerfile}} {{path/to/another/Dockerfile}}`
```

Reply:
`example.com` is good. The port isn't strictly required. The default docker registry port is `5000` (not `500` sorry).

Reply:
Actually, I think it's best if we show an example of each. Just updated it